src/SourceBrowser.Site/Scripts/search.js

```diff
@@ -11,6 +11,25 @@
         //Only bind if the page is searchable. (ie. we're in a repository)
         window.History.Adapter.bind(window, 'statechange', handleStateChange);
     }
+
+    // Drag-to-resize search results pane
+    // Adapted from http://www.catchmyfame.com/2010/08/12/adjustable-columns-with-jquery/
+
+    var stopTwoLeft = parseInt($('#main-content').offset().left) + 300;
```

Comment:
Sure:
*one* refers to the first column (main contents), *two* is the second column (the draggable bar), and *three* refers to the third column which contains the search results (or the tree view).
Now that I look at them, the variable names are not intuitive. My apologies. stopTwoLeft is the left drag boundary (for the second column) and stopTwoRigfht is the right drag boundary. We're allowing dragging such that the main column is at least 300px wide, and the right column is at least 200px wide.

.draggable() makes a jquery element draggable. first, I constrain the dragging to the X-axis.
Second, I provide code to be executed when dragging begins. We set variables leftOneStart and leftThreeStart, which will be used in the following function:
The second function specifies what happens when user drags the element. We are adjusting widths of the first and third column.
Finally, containment specifies boundaries of the drag zone, which I defined earlier.